### PR TITLE
added hibernate annotations

### PR DESCRIPTION
**Pull request by Azadios**

### CODE REVIEW

File: src/main/java/ru/hh/school/entity/Area.java [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 //TODO: оформите entity
+@Entity
+@Table(name = "area")
 public class Area {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "area_id")
   private Integer id;
 
+  @Column(name = "name")
```
Comment (Afftobus) [inline src/main/java/ru/hh/school/entity/Area.java:19]: когда совпадает имя поля класса с именем в базе не обязательно добавлять аннотацию @Column

**Review by Afftobus — COMMENTED** [review]
Всё ОК. 
ДЗ принял.

File: src/main/java/ru/hh/school/entity/Resume.java [hunk 2]
```diff
@@ -15,9 +22,11 @@
...
   @Id
-  @GeneratedValue(/* здесь место для вашего кода */)
+  @SequenceGenerator(name = "resume_id_seq", sequenceName = "resume_id_seq", allocationSize = 10)
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "resume_id_seq")
   private Integer id;
 
+  @Column(name = "description")
   private String description;
 
   Resume() {}
```
Comment (aimiebell) [inline src/main/java/ru/hh/school/entity/Resume.java:32]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Comment (Azadios) [reply src/main/java/ru/hh/school/entity/Resume.java:32]: Have a better idea on naming constructor? :)

**Pull request closed without merging**